src/__tests__/uid.test.tsx

```diff
@@ -1,10 +1,10 @@
-import React from "react"
-
 import uid from "../uid"
 
 describe("unique id", () => {
   const options = 100
+  // @ts-ignore To avoid adding polyfill for `fill` becoz fill is ES6 feature and our target is ES5
   const ids = new Array(options).fill(" ").map(item => uid())
+  // @ts-ignore To avoid adding polyfill for `from`
```

Comment:
To use ES6 features in this file like `fill`, `from`, `Set`, we have to add polyfills. As this is just a test file, not affecting our library consumer in any way. So I added `@ts-ignore` to ignore these `ts` errors.